website/src/utils/yamlDataUtils/update.js

```diff
@@ -68,6 +68,20 @@ export const updateConn = (path, value, yamlData) => {
     const steps = path.split(".")
     const target = getValueAtPath(data, steps.splice(0, 3))
     const conn = target.key
+    if (value.toLowerCase() === 'solr' || value.toLowerCase() === 'neo4j') { 
```

Comment:
if this function is called only when the connection name changes. 
call this function `updateConnectionName`